models/experimental/yolov4/ttnn/downsample1.py

```diff
@@ -45,9 +45,17 @@ def __call__(self, device, input_tensor):
         output_tensor = self.conv7(device, output_tensor)
         output_tensor = ttnn.mish(output_tensor)
 
-        output_tensor = ttnn.sharded_to_interleaved(output_tensor, ttnn.L1_MEMORY_CONFIG)
-        output_tensor_left = ttnn.sharded_to_interleaved(output_tensor_left, ttnn.L1_MEMORY_CONFIG)
-        output_tensor = ttnn.concat([output_tensor, output_tensor_left], dim=3, memory_config=ttnn.L1_MEMORY_CONFIG)
+        output_tensor = ttnn.to_layout(output_tensor, layout=ttnn.ROW_MAJOR_LAYOUT)
+        output_tensor_left = ttnn.to_layout(output_tensor_left, layout=ttnn.ROW_MAJOR_LAYOUT)
+        output_sharded_memory_config = ttnn.create_sharded_memory_config(
+            [512, 128],
```

Comment:
why is the output shard memory config [512, 128] here?

Reply:
Shard shape of the inputs is [512, 64]. It's a width concat of two height sharded with inner dim 64, so inner dim ends up 128.